<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>streamLineDialog</class>
 <widget class="QDialog" name="streamLineDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Streamlines</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ElmerGUI.qrc">
    <normaloff>:/icons/Mesh3D.png</normaloff>:/icons/Mesh3D.png</iconset>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout">
     <item>
      <widget class="QGroupBox" name="controlGroup">
       <property name="title">
        <string>Controls</string>
       </property>
       <layout class="QVBoxLayout">
        <item>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QLabel" name="vectorLabel">
            <property name="text">
             <string>Vector:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="vectorCombo">
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="propagLabel">
            <property name="text">
             <string>Max. time:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="propagationTime">
            <property name="text">
             <string>1.0</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="stepLabel">
            <property name="text">
             <string>Step length:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLineEdit" name="stepLength">
            <property name="text">
             <string>0.05</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="threadsLabel">
            <property name="text">
             <string>Threads:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="threads">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="integStepLabel">
            <property name="text">
             <string>Integ. length:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLineEdit" name="integStepLength">
            <property name="text">
             <string>0.005</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <widget class="QRadioButton" name="useSurfaceGrid">
            <property name="text">
             <string>Use surface mesh</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="forward">
            <property name="text">
             <string>Integrate forwards</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="useVolumeGrid">
            <property name="text">
             <string>Use volume mesh</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QCheckBox" name="backward">
            <property name="text">
             <string>Integrate backwards</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="3">
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="colorGroup">
       <property name="title">
        <string>Color</string>
       </property>
       <layout class="QVBoxLayout">
        <item>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QLabel" name="lengthLabel">
            <property name="text">
             <string>Color:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="colorCombo">
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
           </widget>
          </item>
		  <item>
           <widget class="QLabel" name="nullColorLabel">
            <property name="text">
             <string>â– </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="nullColorButton">
            <property name="text">
             <string>Select color</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="minLabel">
            <property name="text">
             <string>Min:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="minVal">
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="maxLabel">
            <property name="text">
             <string>Max:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLineEdit" name="maxVal">
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QCheckBox" name="keepLimits">
            <property name="text">
             <string>Keep limits</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="appearanceGroup">
       <property name="title">
        <string>Appearance</string>
       </property>
       <layout class="QVBoxLayout">
        <item>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <widget class="QRadioButton" name="drawLines">
            <property name="text">
             <string>Draw lines</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="lineWidthLabel">
            <property name="text">
             <string>Line width:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QSpinBox" name="lineWidth">
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="drawRibbon">
            <property name="text">
             <string>Draw ribbons</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="ribbonWidthLabel">
            <property name="text">
             <string>Ribbon width:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QSpinBox" name="ribbonWidth">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="value">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout">
     <item>
      <widget class="QGroupBox" name="sourceTypeGroup">
       <property name="title">
        <string>Source type</string>
       </property>
       <layout class="QVBoxLayout">
        <item>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <widget class="QRadioButton" name="sphereSource">
            <property name="text">
             <string>Sphere source</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="pickSource">
            <property name="text">
             <string>Pick source point ('p')</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="lineSource">
            <property name="text">
             <string>Line source</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="pointSourceGroup">
       <property name="title">
        <string>Sphere source</string>
       </property>
       <layout class="QVBoxLayout">
        <item>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="xPointStartLabel">
            <property name="text">
             <string>Center X:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="centerX">
            <property name="text">
             <string>0.0</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="radiusLabel">
            <property name="text">
             <string>Radius:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLineEdit" name="radius">
            <property name="text">
             <string>0.0</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="yPointStartLabel">
            <property name="text">
             <string>Center Y:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="centerY">
            <property name="text">
             <string>0.0</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="pointsLabel">
            <property name="text">
             <string>Points:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QSpinBox" name="points">
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="zPointStartLabel">
            <property name="text">
             <string>Center Z:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="centerZ">
            <property name="text">
             <string>0.0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="rakeSourceGroup">
       <property name="title">
        <string>Line source</string>
       </property>
       <layout class="QVBoxLayout">
        <item>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="xStartLabel">
            <property name="text">
             <string>Start X:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="startX">
            <property name="text">
             <string>0.0</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="xEndLabel">
            <property name="text">
             <string>End X:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4" colspan="2">
           <widget class="QLineEdit" name="endX">
            <property name="text">
             <string>0.0</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="yStartLabel">
            <property name="text">
             <string>Start Y:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="startY">
            <property name="text">
             <string>0.0</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="yEndLabel">
            <property name="text">
             <string>End Y:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4" colspan="2">
           <widget class="QLineEdit" name="endY">
            <property name="text">
             <string>0.0</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="zStartLabel">
            <property name="text">
             <string>Start Z:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="startZ">
            <property name="text">
             <string>0.0</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="yEndLabel_2">
            <property name="text">
             <string>End Z:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4" colspan="2">
           <widget class="QLineEdit" name="endZ">
            <property name="text">
             <string>0.0</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="linesLabel">
            <property name="text">
             <string>Points:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="lines">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QCheckBox" name="rake">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Draw source line </string>
            </property>
           </widget>
          </item>
          <item row="4" column="3" colspan="2">
           <widget class="QLabel" name="rakeWidthLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Source line width:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="5">
           <widget class="QSpinBox" name="rakeWidth">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="value">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../ElmerGUI.qrc">
         <normaloff>:/icons/dialog-close.png</normaloff>:/icons/dialog-close.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="icon">
        <iconset resource="../ElmerGUI.qrc">
         <normaloff>:/icons/dialog-ok-apply.png</normaloff>:/icons/dialog-ok-apply.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ElmerGUI.qrc"/>
 </resources>
 <connections/>
</ui>
